photon-server/src/main/java/org/photonvision/vision/processes/VisionSourceManager.java

```diff
@@ -40,11 +38,25 @@
         for (var device : allDevices) {
```

Comment:
Move the `filterAllowedDevices` method down to below the public ones. It doesn't need to be the first in the file